src/radiation/radiation_system.hpp

```diff
@@ -1693,6 +1711,11 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 					amrex::Gpu::Atomic::Add(p_num_failed_coupling_local, 1);
 				}
 
+				// update iteration counter: (+1, +ite, max(self, ite))
+				amrex::Gpu::Atomic::Add(&p_iteration_counter_local[0], 1);
+				amrex::Gpu::Atomic::Add(&p_iteration_counter_local[1], n + 1);
+				amrex::Gpu::Atomic::Max(&p_iteration_counter_local[2], n + 1);
```

Comment:
Can you add a comment explaining what each element of `p_iteration_counter_local` means?